src/Sarif/Converters/ClangAnalyzerConverter.cs

```diff
@@ -9,15 +9,15 @@
 
 namespace Microsoft.CodeAnalysis.Sarif.Converters
 {
-    internal class ClangAnalyzerConverter : IToolFileConverter
+    internal class ClangAnalyzerConverter : ToolFileConverterBase
```

Comment:
@michaelcfanning Now this is interesting: I didn't realize that our built-in converters did _not_ already extend `ToolFileConverterBase`. Apparently `FxCopConverter` and `AndroidStudioConverter` did, and `ClangAnalyzerConverter` and `CppCheckConverter` did not -- because they did not need the "logical location" functionality.